adapters/common/src/nemo_fabric_adapters/common/utils.py

```
@@ -17,6 +17,12 @@
 _FIELD_NAME = re.compile(r"[!#$%&'*+\-.^_`|~0-9A-Za-z]+")
 
 
+def normalize_user_input(value: Any) -> str:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)